### ISSUE TITLE
Script for Celemony Melodyne? 

### ISSUE THREAD

**Issue by alekssamos** [MatejGolian/ReaHotkey#23]
Hi. Is it possible to create any scripts for  Melodyne?
I know about sibiac, but I like it when you grab a note with the mouse, it sounds, and you roll it with the mouse, it doesn't look like that from the keyboard, it's different. Is it possible to use some script, perhaps even on AutoIt, by searching pixels by color and moving the mouse  to the notes that require correction? And then I will physically hold down the left button and gently move it up and down. The problem is just hovering the mouse cursor over the notes.

https://github.com/user-attachments/assets/2755739c-075a-4640-bcc9-24c000a6cd00

**Comment by MatejGolian** [comment]
I don't use Melodine myself, but from the looks of it @Timtam would probably like to include some kind of script for it at some point.
Because I don't use the software I can't tell you what we're able or unable to do, but theoretically yes, what you request may be technically possible. At least it sounds so.
That's all I can say - I don't have enough knowledge to give you more details...

**Comment by Timtam** [comment]
Yeah, Melodyne is on my list of things that I'd love to look into at some point, but don't know when or if it'll happen at all. Melodyne is a rather complex topic. SIBIAC shows that it is possible, and we can do everything that SIBIAC can do too, maybe even more, but it'll take alot of time, not just my time, but also the time of at least another sighted person who has to familiarize with Melodyne first. I at least got a Melodyne Studio license here already. I can't promise anything, it happens when it happens, but i'll keep your issue around so that I remember as soon as i'm working on it.

**Comment by acerbt** [comment]
I can certainly say for certain that a melodyne script would be great. i believe some thought should be made about that sooner rather than later, as NVDA is undergoing an arcatecture shift from 32 to 64 bit, meaning that older add-ons, like sibiac, will be left behind and no longer will work. Either that or those of us that use sibiac and similar things will have to stay on nvda 2025 well after nvda 2026 comes out.

**Comment by MatejGolian** [comment]
@acerbt, I think @Timtam still has plans to do something about it, but as far as I know he's mostly occupied by other kind of duties these days.
I don't personally use Melodine and therefore don't really feel like taking this task on myself. However, if someone skilled in AHK wanted to step in, I can provide help with grasping the concepts behind ReaHotkey itself.
